js/fieldmanager.js

```diff
@@ -1,5 +1,34 @@
 var fm = {};
 
+/**
+ * Polyfill Element.matches, if necessary.
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill
```

Comment:
The link should be `https://developer.mozilla.org/en-US/docs/Web/API/Element/matches#Polyfill`

Reply:
This specific `matches` polyfill was taken from the `closest` polyfill page, which is slightly different from the one present on the `matches` polyfill page you linked above. I'm only including it because `closest` depends on it, so I'm pulling both polyfills from the same location. Good catch, though!